File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/vip_level_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class VipLevelVO:
    """VIP等级响应"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    name: str = None
    level_value: int = None
    required_points: int = None
    description: str = None
    icon: str = None
    badge: str = None

File: sdkwork-app-sdk-csharp/Models/VipLevelVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class VipLevelVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public string? Name { get; set; }
        public int? LevelValue { get; set; }
        public int? RequiredPoints { get; set; }
        public string? Description { get; set; }
        public string? Icon { get; set; }
        public string? Badge { get; set; }
    }
}
